modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -43,7 +43,7 @@ The startup time for a Mule applications is correlated to the amount of vCPU cor
 
 == Application Performance
 
-The amount of resources allocated to your Mule application determin the application's performance. Below are the approximate values for throughput based on the amount of vCPU cores allocated for a single Mule application performing simple processing on a 10 KB payload.
+The resources allocated to your Mule application determine the application's performance. Below are the approximate values for throughput based on the amount of vCPU cores allocated for a single Mule application performing simple processing on a 10 KB payload.
```

Comment:
Below > Following, Here, These... (and end sentence with a colon that points directly to the table, as in the previous table instance)

10 KB payload > 10-KB payload